### DISCUSSION TITLE
I just Update chrome OS via edit the grub file enable _updates. and I ran into a serious BUG Please Help.

### DISCUSSION THREAD

**Discussion by YashAgarwalDev** [sebanc/brunch#709] [General]
Well I just update my device and it updates succesfully . after that I remove the lines Enable_updates to stop auto updates then Reboot. system turn on and again its checking for updates and start downloading. how this is possible if I removed the lines.
Second than after I reboot it stucks on updating brunch framework for 1 hour than nothing happens . I have to remove the Chrome OS . I want to know why this auto update happend and why It stucks on brunch framework page for so long.

Here What I use.
Latest Brunch framework which is recently release ( Brunch r86 k5.4 stable 20201201 )
Rammus 86

Laptop Details.
Dell Latitude E7240 I5 4th gen. with Samsung SSD mSATA ( In laptop Every thing is working including Microphone and touch pad with gestures. only One bug I found that my BT Spearker are connected properly but audio did not transfer in that. any ways I want to know about this bug " Why It keeps updating if lines are removed and stuck on update brunch frame work" 

Suggesions 
Please add any percentage system on brunch boot page while brunch updates so that we may  know its working or get stucked.

**Comment by sebanc** [comment]
You can permanently disable the bootlogo and have the log by removing "console= vt.global_cursor_default=0 brunch_bootsplash=default" in the grub configuration.

I could not reproduce this issue (updated to r87, then removed "enable_updates" option) but it still worked fine and updates were correctly disabled. My guess would be that you broke the grub config somehow.

Btw, you don't specifically need to remove "options=enable_updates" as updates will only be applied when you check for updates.

**Reply by YashAgarwalDev** [reply]
> You can permanently disable the bootlogo and have the log by removing "console= vt.global_cursor_default=0 brunch_bootsplash=default" in the grub configuration.
> 
> I could not reproduce this issue (updated to r87, then removed "enable_updates" option) but it still worked fine and updates were correctly disabled. My guess would be that you broke the grub config somehow.
> 
> Btw, you don't specifically need to remove "options=enable_updates" as updates will only be applied when you check for updates.

When will Brunch R87 will release with lots of fixes.

**Reply by asukumar10248** [reply]
set timeout =0 and console= to disable all the the boot animation

**Comment by YashAgarwalDev** [comment]
I still don't know how grub config broke.

**Comment by YashAgarwalDev** [comment]
is there any way to install it on VirtualBox or VMWare.

**Reply by sebanc** [reply]
No, there is no support for vmware gpu drivers in chromeos
